client/verta/docs/tutorials/deployment/endpoint_canary_update.rst

```diff
@@ -0,0 +1,50 @@
+Updating endpoints with canary
+==============================
+
+Let's say you'd like to deploy a new model, but rather than commit to updating the Endpoint
+`all at once <endpoint_update.html>`__, you'd like to roll out your model slowly and
+incrementally, keeping an eye on its behavior in case things don't go as expected.
```

Comment:
Very informal. It's ok to be a bit informal, but this is very casual.

Reply:
I've reworded it to be a bit more distant